sktime/forecasting/chronos.py

```diff
@@ -0,0 +1,268 @@
+"""Implements Chronos forecaster."""
+
+__author__ = ["Z-Fran", "benheid"]
+# __all__ = ["ChronosForecaster"]
+
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+from skbase.utils.dependencies import _check_soft_dependencies
+
+from sktime.forecasting.base import ForecastingHorizon, _BaseGlobalForecaster
+from sktime.libs.chronos import ChronosPipeline
+
+if _check_soft_dependencies("torch", severity="none"):
+    import torch
+else:
+
+    class torch:
+        """Dummy class if torch is unavailable."""
+
+        bfloat16 = None
+
+
+if _check_soft_dependencies("transformers", severity="none"):
+    import transformers
+
+
+class ChronosForecaster(_BaseGlobalForecaster):
+    """Chronos forecaster.
+
+    Parameters
+    ----------
+    model_path : str
+        Path to the Chronos' huggingface model.
+    config : dict, default={}
+        Configuration to use for the model.
+    seed: int, optional, default=None
+        Random seed for transformers.
+
+    Examples
+    --------
+    >>> from sktime.datasets import load_airline
+    >>> from sktime.forecasting.chronos import ChronosForecaster
+    >>> from sktime.split import temporal_train_test_split
+    >>> from sktime.forecasting.base import ForecastingHorizon
+    >>> y = load_airline()
+    >>> y_train, y_test = temporal_train_test_split(y)
+    >>> fh = ForecastingHorizon(y_test.index, is_relative=False)
+    >>> forecaster = ChronosForecaster("amazon/chronos-t5-tiny")
+    >>> forecaster.fit(y_train)
+    >>> y_pred = forecaster.predict(fh)
+    """
+
+    # tag values are "safe defaults" which can usually be left as-is
+    _tags = {
+        "python_dependencies": ["torch", "transformers"],
```

Comment:
`accelerate` is another dependency here, especially when using the `device_map` explicitly
I think we should add `accelerate` to sktime's `dl` dependency set as well

Reply:
`accelerate` added as a dependency